### PR TITLE
chore: translation of 'Using the Effect Hook' section part1 (#74)

### PR DESCRIPTION
**Pull request by ApolllonDev**
To-Do:
---
- [x]  finish Part 2 translation

Fixes for #74 
---
- [x] еффектами - з одною Ф
- [x] классовими - одна С
- [x] вимагають => потребують

### CODE REVIEW

File: content/docs/hooks-effect.md [hunk 1]
```diff
@@ -1,57 +1,57 @@
 ---
 id: hooks-state
-title: Using the Effect Hook
+title: Використовуємо хук еффекту
```
Comment (liashchynskyi) [inline content/docs/hooks-effect.md]: одна `ф`

File: content/docs/hooks-effect.md [hunk 2]
```diff
@@ -1,57 +1,57 @@
...
 permalink: docs/hooks-effect.html
 next: hooks-rules.html
 prev: hooks-state.html
 ---
 
-*Hooks* are a new addition in React 16.8. They let you use state and other React features without writing a class.
+*Хуки* — це новинка в React 16.8. Вони дозволяють вам використовувати стан та інші можливості React без написання класу.
 
-The *Effect Hook* lets you perform side effects in function components:
+*Хук еффекту* дозволяє вам виконувати побічні еффекти в функціональному компоненті:
```
Comment (liashchynskyi) [inline content/docs/hooks-effect.md]: то саме

Comment (liashchynskyi) [reply content/docs/hooks-effect.md]: і ще багато разів 😹

**Comment by ApolllonDev** [comment]
Unfortunately, I won't be able to work on #74 translation until late September.
Therefore, it would be best if someone can finish it so I don't slow down the team.
- [ ] finish Part 2 translation

**Comment by ApolllonDev** [comment]
@vldmrkl thank you for finishing this one, looks good 🚀

**Comment by vldmrkl** [comment]
@AppolllonDev good to merge?

File: content/docs/hooks-effect.md [hunk 3]
```diff
@@ -63,35 +63,35 @@ class Example extends React.Component {
...
     );
   }
 }
 ```
 
-Note how **we have to duplicate the code between these two lifecycle methods in class.**
+Зверніть увагу, що **ми продублювали код між цими двома методами життєвого циклу в класі.**
 
-This is because in many cases we want to perform the same side effect regardless of whether the component just mounted, or if it has been updated. Conceptually, we want it to happen after every render -- but React class components don't have a method like this. We could extract a separate method but we would still have to call it in two places.
+Це все тому, що в багатьох випадках ми хочемо виконати той самий побічний ефект незалжно від того чи компонент тільки змонтувався, або він оновився. Ми б хотіли, щоб ці побічні ефекти викликались після кожного рендеру, але класові React-компоненти не мають методу, який це може зробити. Ми б могли винести окремий метод, але нам все рівно би потрібно було викликати їх в двох місцях.
```
Comment (ApolllonDev) [inline content/docs/hooks-effect.md:88]:
```suggestion
Це все тому, що в багатьох випадках ми хочемо виконати той самий побічний ефект незалжно від того чи компонент тільки змонтувався, або він оновився. Ми б хотіли, щоб ці побічні ефекти викликались після кожного рендеру, але класові React-компоненти не мають методу, який це може зробити. Ми б могли винести окремий метод, але нам все рівно би потрібно було викликати їх у двох місцях.
```

File: content/docs/hooks-effect.md [hunk 4]
```diff
@@ -63,35 +63,35 @@ class Example extends React.Component {
...
 
-Now let's see how we can do the same with the `useEffect` Hook.
+Тепер, давайте розглянемо те, як ми можемо реалізувати це саме, використовуючи хук `useEffect`.
```
Comment (ApolllonDev) [inline content/docs/hooks-effect.md:90]:
```suggestion
Тепер, давайте розглянемо те, як ми можемо реалізувати теж саме, використовуючи хук `useEffect`.
```

**Pull request merged**